test/ocspapitest.c

```diff
@@ -78,7 +78,10 @@ static OCSP_BASICRESP *make_dummy_resp(void)
     ASN1_BIT_STRING *key = ASN1_BIT_STRING_new();
     ASN1_INTEGER *serial = ASN1_INTEGER_new();
 
-    if (!X509_NAME_add_entry_by_NID(name, NID_commonName, MBSTRING_ASC,
+    if (!TEST_ptr(name)
+        || !TEST_ptr(key)
+        || !TEST_ptr(serial)
+        || !X509_NAME_add_entry_by_NID(name, NID_commonName, MBSTRING_ASC,
```

Comment:
Any chance converting this and the two following lines to use TEST_true so the exact failure is reported?

Reply:
Changed.